### PR TITLE
Upload code coverage to Codacy

### PR DESCRIPTION
**Pull request by gregfletch**
*Related Defect(s), Issue(s) or Task(s)*

*Why?*

We want to try to upload code coverage results to Codacy in addition to CodeCov.

*How?*

Add codacy coverage gem and add codacy to coverage config.

*How did this defect occur?*

N/A

*Risks*

Low

*Requested Reviewers*

List the developers that should review this pull request.

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -24,6 +26,8 @@
 
 Dir[File.join(Dir.pwd, 'lib', 'token_validator.rb')].sort.each { |file| require file }
 
+WebMock.disable_net_connect!(allow: 'https://api.codacy.com')
```
Comment (dragoszt) [inline spec/spec_helper.rb]: Why is this needed?

Comment (gregfletch) [reply spec/spec_helper.rb]: So that we can send the coverage report to Codacy without the network request being blocked by WebMock. The codecov gem disables WebMock and VCR before sending coverage reports, but the Codacy coverage gem does not do this, so we have to manually disable WebMock and/or VCR manually.
